docs/guides/mfa/25_lookup_secrets.mdx

```diff
@@ -0,0 +1,94 @@
+---
+id: lookup_secrets
+title: Lookup Secrets / Recovery Codes
+sidebar_title: Lookup Secrets
```

Comment:
```suggestion
title: Lookup Secrets (Recovery Codes)
```
How about we handle this similarly to the TOTP case? Our name (which we use in the UI) goes first, the name that might help some ppl identify the tech. is in parenthesis. We get rid of `sidebar_title` in the name of having lighter metadata.